src/common/blkdev.cc

```diff
@@ -410,20 +410,19 @@ std::string get_device_id(const std::string& devname)
     return device_id;
   }
 
-  // either udev_device_get_property_value() failed, or succeeded but returned nothing;
-  // trying to read from files.
+  // either udev_device_get_property_value() failed, or succeeded but
+  // returned nothing; trying to read from files.  note that the 'vendor'
+  // file rarely contains the actual vendor; it's usually 'ATA'.
```

Comment:
On my host, the vendor is HP.

erwan@mr-meeseeks:~$ cat /sys/block/sda/device/vendor 
HP

So as long it could provide some info it would worth keeping it as it might encode information.